cylc/flow/exceptions.py

```diff
@@ -430,7 +430,6 @@ class NoHostsError(CylcError):
     """None of the hosts of a given platform were reachable."""
     def __init__(self, platform):
         self.platform_name = platform['name']
-        super().__init__()
```

Comment:
Ok, funky.

It looks like `HostSelectException` might be similar?

Reply:
Good spot, I have fixed that too now